tests/test_pipeline.py

```diff
@@ -259,3 +259,15 @@ def test_delete_pipeline_v6(client_version_6, requests_mock):
         json={"payload": None, "errorMessages": []},
     )
     pipeline.delete()
+
+
+def test_collection_process_complete(client_version_6, requests_mock):
+    pipeline = Pipeline(Project(client_version_6, "LoadTesting"), "discharge")
+    requests_mock.post(
+        f"{URL_BASE_ID}/rest/experimental/textanalysis/projects/"
+        f"LoadTesting/pipelines/discharge/collectionProcessComplete",
+        headers={"Content-Type": "application/json"},
+        json={"payload": None, "errorMessages": []},
+        status_code=204
```

Comment:
If the backend was changed to return a 200, then this test should also be changed to return a 200.

Reply:
Yes thank you for fixing it!